superset-frontend/src/explore/components/DisplayQueryButton.jsx

```diff
@@ -75,6 +75,24 @@ const CopyButton = styled(Button)`
   && {
     margin-left: ${({ theme }) => theme.gridUnit * 2}px;
   }
+
+  i {
+    padding: 0;
+  }
+`;
+
+const CopyButtonViewQuery = styled(Button)`
+  padding: ${({ theme }) => theme.gridUnit / 2}px
```

Comment:
Does it look good (or even better?) if it's just 2 gridUnits of padding all the way around? Super minor detail, but I'm trying to avoid fractional gridUnits, in case we decide the gridUnit should be 5px, and we find ourselves in weird half-pixel-land.